configs/openmrs/initializer_config/ampathforms/historique_de_sante.json

```diff
@@ -618,7 +618,7 @@
                       "id": "smokingDuration(Year(S))",
                       "questionOptions": {
                         "rendering": "number",
-                        "concept": "159931AAAAAAAAAAAAAAAAAAAAAAAAAAAAAA",
+                        "concept": "dbab4772-1814-466f-a4ff-22aceca57200",
```

Comment:
You've changed the concept's UUID but missed updating the label, i.e. instead of being `Smoking Duration (Year(s))` it should be `Smoking Duration`?

Reply:
I have updated all manually appended label units.
This should be done by the RFE, and I created a [ticket](https://openmrs.atlassian.net/browse/O3-4201) for this.